crates/libtiny_tui/src/config.rs

```diff
@@ -26,6 +27,13 @@ pub struct Style {
     pub bg: u16,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub(crate) enum UiStyle {
+    Compact,
+    Aligned { max_nick_length: usize },
```

Comment:
tiny is supposed to easy to configure and provide good UX out of the box, so I think making `max_nick_length` optional would be good. I think I should be able to change

```
ui_style: compact
```

to

```
ui_style: aligned
```

and it should work.

Perhaps moving `max_nick_length` to another top-level field might make sense. Something like:

```
ui_style: aligned
max_nick_length: 12
```